src/madhavi_Raut/Assignment_30_to_40/Assignment_30/FindSecondMaxNumber.java

```diff
@@ -0,0 +1,24 @@
+package madhavi_Raut.Assignment_30_to_40.Assignment_30;
+
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+public class FindSecondMaxNumber {
+
+	int getSecondMaxNumber(int[] numArr) {
+		int maxNum = numArr[0], secondMaxNum = 0;
+		for (int index = 0; index < numArr.length; index++) {
```

Comment:
this logic fails for below input @MadhaviRTechnoCredits 
int[] arr = { 112, 93, 10, 33, 43, 111, 55, 97, 11, 3, 88, 99 };

Have reference of this code
<img width="405" alt="Screenshot 2021-05-04 at 8 49 46 PM" src="https://user-images.githubusercontent.com/58640315/117028341-4186af00-ad1b-11eb-874a-45ec88c891c4.png">